src/Swift.Bindings/src/Demangler/TbdParser/Logging/ConsoleLogger.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+
+namespace TbdParser.Logging
+{
+    /// <summary>
+    /// Simple console logger implementation
+    /// </summary>
+    public class ConsoleLogger : ILogger
```

Comment:
Can we add this to be project-wide? There is -v flag but it doesn't implement any logger.

Reply:
> There is -v flag but it doesn't implement any logger.

Yes, currently we are passing `int verbosity` around and based on the value we print to the console. 

If this format would be appropriate, there shouldn't be an issue to port it for the entire codebase. I would probably make it a separate PR to keep this PR smaller as it already has ~1000 line changes.

Reply:
Could we add this to a tracking issue? I also think this would be great to have on the entire codebase!